python/tvm/relay/op/contrib/ethosu.py

```diff
@@ -1671,7 +1671,18 @@ def check_compatible_size(mode, method, upscale_size, ifm_size):
             return False
         if self.method not in ("nearest_neighbor", "linear"):
             return False
-        if self.coordinate_transformation_mode not in ("asymmetric", "align_corners"):
+        if self.coordinate_transformation_mode not in (
+            "asymmetric",
+            "align_corners",
+            "half_pixel",
+        ):
+            return False
+        if (
+            self.coordinate_transformation_mode == "half_pixel"
+            and self.rounding_method != "round_prefer_ceil"
+        ):
+            return False
+        if self.coordinate_transformation_mode != "half_pixel" and self.rounding_method != "":
```

Comment:
Perhaps to avoid confusion we can combine this condition into the previous one as they're related?